### PR TITLE
Add module command

### PR DESCRIPTION
**Pull request by ripleyaffect**
- takes the module specified with `--name` and adds the checkpoints from the library to it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,13 +51,32 @@ OPTIONS
...
-  all `.yaml` files in the module and program directories and adds a new uuid
-  to any file without one.
+  Loads all the `content.md` files in the library directory and all `.yaml`
+  files in the module and program directories and adds a new uuid to any file
+  without one.
 ```
 
 _See code: [src/commands/uuids.js](https://github.com/Thinkful/zapdos/blob/v0.0.1/src/commands/uuids.js)_
 
+## `zapdos uuids`
```
Comment (learykara) [inline README.md]: should this say `zapdos module`?

File: README.md [hunk 2]
```diff
@@ -51,13 +51,32 @@ OPTIONS
...
+
+Generate a module
+
+```
+USAGE
+  $ zapdos module --name [name]
```
Comment (learykara) [inline README.md]: not going with `noun:verb`? or is this something different

File: src/lib/getLibraryFile.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    const fileRaw = await fs.readFile(filePath, 'utf8');
+
+    // Add required dashes to front of file if missing
+    const fileCleaned = /^---\n/.test(fileRaw) ? fileRaw : `---\n${fileRaw}`;
+
+    // Create a lightweight object with path and data
+    return {
+      ...frontMatter(fileCleaned),
+      path: filePath,
+      src: filePath.match(/\/([a-z0-9\.\_\+\-]+)\/content\.md$/i)[1],
```
Comment (learykara) [inline src/lib/getLibraryFile.js]: guaranteed to exist?

File: README.md (hunk 2 above)
Comment (ripleyaffect) [reply README.md]: going w/ `verb:noun`, but decided after making this. this command will be moved to `build:module`

**Pull request merged**